test/DynamoCoreWpfTests/ConverterTests.cs

```diff
@@ -20,12 +20,6 @@ namespace Dynamo.Tests
 {
     class ConverterTests
     {
-        [SetUp]
-        public void Setup()
-        {
-            DynamoModel.IsTestMode = true;
```

Comment:
This is not unrelated. This was put in to ensure that all tests ran `IsTestMode` because that flag was dictating whether the view was created. That flag is no longer dictating whether the view is created, so it is removed.